<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsEditorWidget</class>
 <widget class="QWidget" name="SettingsEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Текстовые метки */
#topPanel {
	border-top-width: 1px;
	border-top-style: solid;
	border-top-color: #5f5f5f;
	border-right-width: 1px;
	border-right-style: solid;
	border-right-color: #5f5f5f;
	background: #282d31;
	color: #d5d6d6;
	padding-top: 2px;
	padding-bottom: 2px;
	min-height: 21px;
}

QListView {
	border: 1px solid #1c1f21;
	border-right-width: 0px;
}

QStackedWidget {
	border: 1px solid #1c1f21;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QListWidget" name="categories">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Text Editor</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/Resources.qrc">
        <normaloff>:/Graphics/Icons/settings_text_editor.png</normaloff>:/Graphics/Icons/settings_text_editor.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chronometry</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/Resources.qrc">
        <normaloff>:/Graphics/Icons/settings_chronometry.png</normaloff>:/Graphics/Icons/settings_chronometry.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="topPanel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="categoriesWidgets">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid lightgray;
    margin-top: 0ex; /* leave space at the top for the title */
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="textEditorPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>600</width>
           <height>498</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="spellChecking">
         <property name="text">
          <string>Spell Checking</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="chronometryPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QRadioButton" name="pagesChronometry">
         <property name="text">
          <string>Pages Chronometry</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="pagesChronometryGroup">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Page</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="pagesChronometrySeconds">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Seconds</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QRadioButton" name="charactersChronometry">
         <property name="text">
          <string>Characters Chronometry</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="charactersChronometryGroup">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="charactersChronometryCharacters">
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Characters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="charactersChronometrySeconds">
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Seconds</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QRadioButton" name="configurableChronometry">
         <property name="text">
          <string>Configurable Chronometry</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="configurableChronometryGroup">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>340</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="7">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="3" colspan="2">
           <widget class="QDoubleSpinBox" name="configurableChronometrySecondsForParagraphDialog">
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Time And Place</string>
            </property>
           </widget>
          </item>
          <item row="3" column="8">
           <widget class="QDoubleSpinBox" name="configurableChronometrySecondsPer50CharactersDialog">
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="8">
           <widget class="QDoubleSpinBox" name="configurableChronometrySecondsPer50CharactersTimeAndPlace">
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="4">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Seconds for
Paragraph</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QDoubleSpinBox" name="configurableChronometrySecondsForParagraphAction">
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>+</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="6">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>+</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="6">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>+</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Dialog</string>
            </property>
           </widget>
          </item>
          <item row="2" column="8">
           <widget class="QDoubleSpinBox" name="configurableChronometrySecondsPer50CharactersAction">
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="7" colspan="3">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Seconds per
50 characters</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Action</string>
            </property>
           </widget>
          </item>
          <item row="1" column="9">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3" colspan="2">
           <widget class="QDoubleSpinBox" name="configurableChronometrySecondsForParagraphTimeAndPlace">
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="2">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>244</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>categories</tabstop>
  <tabstop>spellChecking</tabstop>
  <tabstop>pagesChronometry</tabstop>
  <tabstop>pagesChronometrySeconds</tabstop>
  <tabstop>charactersChronometry</tabstop>
  <tabstop>charactersChronometryCharacters</tabstop>
  <tabstop>charactersChronometrySeconds</tabstop>
  <tabstop>configurableChronometry</tabstop>
  <tabstop>configurableChronometrySecondsForParagraphTimeAndPlace</tabstop>
  <tabstop>configurableChronometrySecondsPer50CharactersTimeAndPlace</tabstop>
  <tabstop>configurableChronometrySecondsForParagraphAction</tabstop>
  <tabstop>configurableChronometrySecondsPer50CharactersAction</tabstop>
  <tabstop>configurableChronometrySecondsForParagraphDialog</tabstop>
  <tabstop>configurableChronometrySecondsPer50CharactersDialog</tabstop>
 </tabstops>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
